src/routes/$repo.$.tsx

```diff
@@ -201,7 +202,7 @@ function UpdateNotifier() {
 }
 
 export default function Repo() {
-  const data = useLoaderData<RepoData>()
+  const data = useLoaderData<typeof loader>()
   const { analyzerData, gitTruckInfo } = data
```

Comment:
can this be typed?

Reply:
Oh, I don't understand why this is any... I will investigate!